src/main/java/edu/oakland/helper/admin/TrackData.java

```diff
@@ -123,6 +123,20 @@ public float getSpeed() {
     return points;
   }
 
+  @Override
```

Comment:
I would add the following Javadoc here:
```suggestion
  /**
   * Creates a print-friendly string of the TrackData data.
   *
   * @return The print friendly string of the TrackData data.
   */
  @Override
```